common/src/main/java/com/wynntils/mc/render/Texture.java

```diff
@@ -14,6 +14,7 @@ public enum Texture {
     ITEM_LOCK("item_lock.png", 16, 16),
     LOOTRUN_LINE("path_arrow.png", 16, 16),
     MANA_BAR("bars_mana.png", 256, 256),
+    GEAR_ICONS("gear_icons.png", 64, 64),
```

Comment:
Keep the alphabetical order please.

Reply:
I tried to order it on file name, then got confused what ordering was used 😅